src/NzbDrone.Core/Download/Clients/LibTorrent/Models/LibTorrentInfoHash.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NzbDrone.Core.Download.Clients.LibTorrent.Models
+{
+    // TODO: Figure out correct json serilization of `null` values, currently setting them to ""
+    public class LibTorrentInfoHash
+    {
+        #nullable enable
+        public string? Hash { get; set; }
+        public string? Status { get; set; }
+
+        public LibTorrentInfoHash(string? hash, string? status)
+        {
+            Hash = hash;
+            Status = status;
+        }
+
+        public LibTorrentInfoHash(IList<string> list)
+        {
+            if (list == null)
+            {
+                throw new ArgumentNullException(nameof(list));
+            }
+
+            Hash = list[0];
+            Status = list[1];
+        }
+
+        public ICollection<string> ToList()
+        {
+            var hash = string.IsNullOrEmpty(Hash) ? "" : Hash;
+            var status = string.IsNullOrEmpty(Status) ? "" : Status;
+            string[] ret = { hash, status };
+            return ret;
+        }
```

Comment:
This is easier to read and less lines.

```suggestion
            var hash = Hash ?? string.Empty;
            var status = Status ?? string.Empty;

            return new[] { hash, status };
        }
```